### PR TITLE
Add a server method mechanism for getting data also.

### PR DESCRIPTION
**Pull request by briandela**
@danielb2 I some how managed to completely mess up my other fork (I'm still scratching my head over that) so I had to fork again. Hence the new PR (I closed the other one) - sorry for that.

I've updated it to expose a function using `server.expose` (instead of the server method in #2)

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -11,40 +12,51 @@ var internals = {
...
+    var payload = {
+        server: {
+            node: process.version,
+            hapi: server._sources[0].server.version,
+            host: server.info.host,
+            port: server.info.port,
+            uri: server.info.uri
+        },
+        plugins: []
+    };
```
Comment (danielb2) [inline lib/index.js]: newline after object (style)

File: lib/index.js [hunk 2]
```diff
@@ -11,40 +12,51 @@ var internals = {
...
+
+
 exports.register = function (server, options, next) {
 
     var validation = Joi.validate(options, internals.schema);
     Hoek.assert(validation.error === null, 'Invalid option(s)');
     options = validation.value;
 
+    server.expose({ payload: internals.generateHapiInfo });
+    server.method('generateHapiInfo', internals.generateHapiInfo, { callback: false });
```
Comment (danielb2) [inline lib/index.js]: I don't think we need to use both.  Expose should be enough :)

Also, makes sure expose outputs what you expect (using a test).

Comment (danielb2) [inline lib/index.js]: Example solution: [here]

[here]: https://gist.github.com/danielb2/69827c33cdfb957815ad

File: package.json [hunk 3]
```diff
@@ -11,6 +11,9 @@
     "url": "https://github.com/danielb2/hapi-info"
   },
   "author": "Daniel Bretoi",
+  "contributors": [
```
Comment (danielb2) [inline package.json]: We're not adding contributors to any of the other hapi-plugins in the official repo, so let's leave this out

File: lib/index.js (hunk 1 above)
Comment (briandela) [inline lib/index.js]: Apologies for missing - I had just copy-pasted from below.

File: package.json (hunk 3 above)
Comment (briandela) [inline package.json]: done.

File: lib/index.js (hunk 2 above)
Comment (briandela) [inline lib/index.js]: Done (also renamed to info)

**Comment by briandela** [comment]
Let me know if you want me to squash the commits before merging.

**Comment by danielb2** [comment]
Thank you :)

**Pull request merged**